source/nodejs/adaptivecards/src/logging/log-enums.ts

```diff
@@ -0,0 +1,7 @@
+export enum LogLevel {
+	Warn,
+	Error,
+	Verbose,
+	Info,
+	Event
```

Comment:
What is the difference between Info and Event?

Reply:
ERror/Verbose/Info are operational traces; Events are more schematized data representing feature 'metrics'

Reply:
How do we actually use those differences when we pull data from the logs? Would anyone who's looking at a report from that data ever want to filter on verbose vs info vs event and what would they get out of it?

Reply:
What is the guidance for third party developers who implement a custom element/action on which type of log to add?

Reply:
Same as they would do today in their regular development workflow: 
Think Info ==> additional context that a developer would like to have access to in case they have to debug something tricky; Doesnt get uploaded by default from the client machine to the servers. 
Warning ==> imp context when functionality is impaired potentially causing errors or compromising functionality downstack but not enough to abort operation/wrkflow at that point in the execution phase
Errors ==> logic errors which we've decided to abort processing for and flag back upto the caller.
Events ==> Events that help instrument pieces of the code in a bit to answer some questions identified upfront by the  developer/feature crew for the control

Reply:
I get why we'd want Error, Warning and Info. I fail to see why we also need Verbose and Event. These might be useful when instrumenting and debugging your own code, but here the goal is for our code (e.g. the AC library) to log things in a way that can be useful for any developer. Where would we use the Verbose/Info/Event levels in the library?